### PR TITLE
added solution for workshop 3 in lecture 2

### PR DESCRIPTION
**Pull request by frANTicBee666**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    package="com.example.solutions">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.MyApplication">
+        <activity android:name=".ws3.SecondActivity"></activity>
```
Comment (emartynov) [inline app/src/main/AndroidManifest.xml:12]: Collapse empty tag?

File: app/src/main/java/com/example/solutions/ws3/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        intent.putExtra(TRANSMITTED_INT, transmittedInt)
+
+        val transmittedBoolean = false
+        intent.putExtra(TRANSMITTED_BOOLEAN, transmittedBoolean)
+
+        // Старт второй активити
+        startActivity(intent)
+    }
+
+    companion object {
```
Comment (emartynov) [inline app/src/main/java/com/example/solutions/ws3/MainActivity.kt:46]: Why do we repeat constants in both activities?

File: app/src/main/java/com/example/solutions/ws3/SecondActivity.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        setContentView(R.layout.activity_second)
+
+        val transmittedString = intent.getStringExtra(TRANSMITTED_STRING)
+
+        val transmittedInt = intent.getIntExtra(TRANSMITTED_INT, -1)
+
+        val transmittedBoolean = intent.getBooleanExtra(TRANSMITTED_BOOLEAN, false)
+
+        textView = findViewById(R.id.first_activity_text_view)
+        textView?.setText("transmittedString $transmittedString, transmittedInt $transmittedInt, transmittedBoolean $transmittedBoolean")
```
Comment (emartynov) [inline app/src/main/java/com/example/solutions/ws3/SecondActivity.kt:25]:
```suggestion
        textView?.text = "Transmitted string: $transmittedString, transmitted int: $transmittedInt, transmitted boolean: $transmittedBoolean"
```

**Review by emartynov — COMMENTED** [review]
The rest LGTM

**Comment by emartynov** [comment]
@frANTicBee666 Ah, wait - it should not be merge to the main branch. It should stay in the solution branch. @PShchahelski, please confirm!

**Comment by PShchahelski** [comment]
> @frANTicBee666 Ah, wait - it should not be merge to the main branch. It should stay in the solution branch. @PShchahelski, please confirm!
Yes, right. we should not merge it. Created another PR.

**Pull request closed without merging**